### PR TITLE
[zh-cn] Translate docs/reference/instrumentation/cri-pod-container-metrics.md 

### PR DESCRIPTION
**Pull request by qlijin**
Fix issue #40415

### CODE REVIEW

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Kubernetes lets you configure the collection of pod and container
+metrics via the {{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI). You
+must enable the `PodAndContainerStatsFromCRI` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) and
+use a compatible CRI implementation (containerd >= 1.6.0, CRI-O >= 1.23.0) to
+use the CRI based collection mechanism.
+-->
+[kubelet](/docs/reference/command-line-tools-reference/kubelet/) 通过 [cAdvisor](https://github.com/google/cadvisor) 收集 pod 和容器指标。
+作为 alpha 功能，Kubernetes 允许你通过{{< glossary_tooltip term_id="cri" text="容器运行时接口">}}（CRI）配置 pod 和容器指标的集合。
+要使用基于 CRI 的收集机制，你必须启用 `PodAndContainerStatsFromCRI` [特性门控](/docs/reference/command-line-tools-reference/feature-gates/)
+并使用兼容的 CRI 实现（containerd >= 1.6.0, CRI-O >= 1.23.0）。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:33]:
```suggestion
[kubelet](/zh-cn/docs/reference/command-line-tools-reference/kubelet/) 通过
[cAdvisor](https://github.com/google/cadvisor) 收集 Pod 和容器指标。作为一个 Alpha 特性，
Kubernetes 允许你通过{{< glossary_tooltip term_id="cri" text="容器运行时接口">}}（CRI）
配置如何收集 Pod 和容器指标。要使用基于 CRI 的收集机制，你必须启用 `PodAndContainerStatsFromCRI`
[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
并使用兼容的 CRI 实现（containerd >= 1.6.0, CRI-O >= 1.23.0）。 
```

**Review by windsonsea — COMMENTED** [review]
请参阅[中文本地化样式指南](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/)：

- 长句适当折行
- url 有中文时添加 `/zh-cn`
- 2 级标题可以加个锚点

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+## CRI Pod & Container Metrics
+
+With `PodAndContainerStatsFromCRI` enabled, the kubelet polls the underlying container
+runtime for pod and container stats instead of inspecting the host system directly using cAdvisor.
+The benefits of relying on the container runtime for this information as opposed to direct
+collection with cAdvisor include:
+-->
+## CRI Pod 和容器指标
+
+当启用 `PodAndContainerStatsFromCRI` 时，kubelet 轮询底层容器运行时以获取 pod 和容器统计信息，而不是直接使用 cAdvisor 检查主机系统。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:47]:
```suggestion
当启用 `PodAndContainerStatsFromCRI` 时，kubelet 轮询底层容器运行时以获取
Pod 和容器统计信息，而不是直接使用 cAdvisor 检查主机系统。
```

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+同直接使用 cAdvisor 收集信息相比，依靠容器运行时获取这些信息的好处包括：
+
+<!--
+- Potential improved performance if the container runtime already collects this information
+  during normal operations. In this case, the data can be re-used instead of being aggregated
+  again by the kubelet.
+-->
+- 如果容器运行时在正常操作中已经收集了这些信息，那么性能就有可能得到提高。在这种情况下，这些数据可以被重用，而不是由 kubelet 再次进行聚合。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:55]:
```suggestion
- 如果容器运行时在正常操作中已经收集了这些信息，那么性能就有可能得到提高。
  在这种情况下，这些数据可以被重用，而不是由 kubelet 再次进行聚合。
```

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+<!--
+- It further decouples the kubelet and the container runtime allowing collection of metrics for
+  container runtimes that don't run processes directly on the host with kubelet where they are
+  observable by cAdvisor (for example: container runtimes that use virtualization).
+-->
+- 它进一步解耦了 kubelet 和容器运行时，允许收集容器运行时的指标，不直接在带有 kubelet 的主机上运行进程，在主机上这些指标可以被 cAdvisor 观测到
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:62]:
```suggestion
- 它进一步解耦了 kubelet 和容器运行时，允许收集容器运行时的指标，
  不直接在可以被 cAdvisor 观测到的带有 kubelet 的主机上运行进程
```

Comment (tengqm) [inline content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:62]:
```suggestion
- 这种做法进一步解耦了 kubelet 和容器运行时。
  对于使用 kubelet 来在主机上运行进程的容器运行时，其行为可用 cAdvisor 观测；
  对于其他运行时（例如，使用虚拟化的容器运行时）而言，这种做法提供了允许收集容器运行时指标的可能性。
```

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+- 如果容器运行时在正常操作中已经收集了这些信息，那么性能就有可能得到提高。
+在这种情况下，这些数据可以被重用，而不是由 kubelet 再次进行聚合。
+
+<!--
+- It further decouples the kubelet and the container runtime allowing collection of metrics for
+  container runtimes that don't run processes directly on the host with kubelet where they are
+  observable by cAdvisor (for example: container runtimes that use virtualization).
+-->
+- 它进一步解耦了 kubelet 和容器运行时，允许收集容器运行时的指标，
+不直接在可以被 cAdvisor 观测到的带有 kubelet 的主机上运行进程
```
Comment (tengqm) [inline content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:65]: Please check my previous comment to this translation.

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+pod 和容器统计信息，而不是直接使用 cAdvisor 检查主机系统。同直接使用 cAdvisor
+收集信息相比，依靠容器运行时获取这些信息的好处包括：
+
+<!--
+- Potential improved performance if the container runtime already collects this information
+  during normal operations. In this case, the data can be re-used instead of being aggregated
+  again by the kubelet.
+-->
+- 如果容器运行时在正常操作中已经收集了这些信息，那么性能就有可能得到提高。
+在这种情况下，这些数据可以被重用，而不是由 kubelet 再次进行聚合。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:57]:
```suggestion
  在这种情况下，这些数据可以被重用，而不是由 kubelet 再次进行聚合。
```

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md (hunk 5 above)
Comment (qlijin) [reply content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:65]: OK

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md (hunk 1 above)
Comment (windsonsea) [reply content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md]: You can directly copy/paste the suggestions from a reviewer if the changes are not easy to identify.

Now, you clicked the `Resolved conversation` button but didn't accept suggestions...... -_!

**Comment by qlijin** [comment]
There is some problem about building. I'll check it

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+The [kubelet](/docs/reference/command-line-tools-reference/kubelet/) collects pod and
+container metrics via [cAdvisor](https://github.com/google/cadvisor). As an alpha feature,
+Kubernetes lets you configure the collection of pod and container
+metrics via the {{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI). You
+must enable the `PodAndContainerStatsFromCRI` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) and
+use a compatible CRI implementation (containerd >= 1.6.0, CRI-O >= 1.23.0) to
+use the CRI based collection mechanism.
+-->
+[kubelet](/zh-cn/docs/reference/command-line-tools-reference/kubelet/) 通过
+[cAdvisor](https://github.com/google/cadvisor) 收集 pod 和容器指标。作为一个 alpha 特性，
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:29]:
```suggestion
[cAdvisor](https://github.com/google/cadvisor) 收集 Pod 和容器指标。作为一个 Alpha 特性，
```

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+Kubernetes 允许你通过{{< glossary_tooltip term_id="cri" text="容器运行时接口">}}（CRI）
+配置收集 pod 和容器指标。要使用基于 CRI 的收集机制，你必须启用 `PodAndContainerStatsFromCRI`
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:31]:
```suggestion
配置收集 Pod 和容器指标。要使用基于 CRI 的收集机制，你必须启用 `PodAndContainerStatsFromCRI`
```

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+With `PodAndContainerStatsFromCRI` enabled, the kubelet polls the underlying container
+runtime for pod and container stats instead of inspecting the host system directly using cAdvisor.
+The benefits of relying on the container runtime for this information as opposed to direct
+collection with cAdvisor include:
+-->
+## CRI Pod 和容器指标   {#cri-pod-container-metrics}
+
+当启用 `PodAndContainerStatsFromCRI` 时，kubelet 轮询底层容器运行时以获取
+pod 和容器统计信息，而不是直接使用 cAdvisor 检查主机系统。同直接使用 cAdvisor
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:48]:
```suggestion
Pod 和容器统计信息，而不是直接使用 cAdvisor 检查主机系统。同直接使用 cAdvisor
```

File: content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+收集信息相比，依靠容器运行时获取这些信息的好处包括：
+
+<!--
+- Potential improved performance if the container runtime already collects this information
+  during normal operations. In this case, the data can be re-used instead of being aggregated
+  again by the kubelet.
+-->
+- 如果容器运行时在正常操作中已经收集了这些信息，那么性能就有可能得到提高。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/instrumentation/cri-pod-container-metrics.md:56]:
```suggestion
- 潜在的性能改善，即如果容器运行时在正常操作中已经收集了这些信息。
```
这样貌似更容易承上启下

**Review by windsonsea — COMMENTED** [review]
/approve

**Pull request merged**